src/context/SWR.tsx

```diff
@@ -15,6 +15,12 @@ interface IMiddleware {
     logger: any;
 }
 
+export const singleCallSettings = {
+    revalidateIfStale: false,
+    revalidateOnFocus: false,
+    revalidateOnReconnect: false,
+};
```

Comment:
Sometimes making a very rare - but sometimes we'll need to make a single server call. Right now this is only used for checking the access grants when logging in.